Package.swift

```diff
@@ -23,7 +23,8 @@ let package = Package(
         .target(
             name: "JWTDecode",
             dependencies: [],
-            path: "JWTDecode"),
+            path: "JWTDecode",
+            exclude: ["Info.plist"]),
```

Comment:
Swift tools version 5.3 added support for bundling resource files. If there are non-code files that are not being bundled, it will produce a warning. So we need to exclude those files.
See https://developer.apple.com/documentation/swift_packages/bundling_resources_with_a_swift_package